lib/emscripten/src/syscalls/unix.rs

```diff
@@ -72,8 +76,38 @@ use libc::{
     F_SETFD,
     SOL_SOCKET,
     TIOCGWINSZ,
+    TIOCSPGRP,
+    // TCGETS,
+    // TCSETSW,
 };
 
+const TCGETS: u64 = 0x5401;
+const TCSETSW: u64 = 0x5403;
```

Comment:
These are not needed since they are the platform dependent translated values, not the wasm ones. Remove and use the ones from libc.

Reply:
They are not exposed in macOS, that's why I had to do it this way.

Reply:
Ok :+1: Maybe add a comment showing this corner case?